### DISCUSSION TITLE
Error connecting to a new deployment - code 100005

### DISCUSSION THREAD

**Discussion by sanketsynthexailabs** [sqlitecloud/discussions#31] [Bug Report] [answered]
I get this error when trying to connected from Python SDK. I just signed up for the service.

conn = sqlitecloud.connect("connection-string-from-the-ui")

sqlitecloud://cmfp7sqxiz.sqlite.cloud:8860?apikey=hiddenforsecurity

SQLiteCloudException: ('An error occurred while reading data from the socket.', <SQLITECLOUD_INTERNAL_ERRCODE.NETWORK: 100005>)

**Comment by marcobambini** [comment]
Hi @sanketsynthexailabs, can you please try to connect again?
We performed several architecture upgrades yesterday, and the services were interrupted several times.
I apologize for any trouble we caused you.

**Comment by sanketsynthexailabs** [comment, marked as answer]
Hi @marcobambini it's working now. Thanks for checking it out. 
What's the timeline to go from Beta to GA?

**Reply by marcobambini** [reply]
GA is planned by the end of September.
